backend/migrations/00006_add_libraries_tables.sql

```diff
@@ -0,0 +1,26 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE public.libraries (
+    id SERIAL PRIMARY KEY,
+    open_content_provider_id integer NOT NULL,
+    external_id VARCHAR,
+    name VARCHAR(255) NOT NULL,
+    language VARCHAR(255),
+    description TEXT,
+    url VARCHAR NOT NULL,
+    image_url VARCHAR,
+    visibility_status BOOLEAN NOT NULL DEFAULT false,
+    created_at timestamp with time zone,
+    updated_at timestamp with time zone,
+    deleted_at timestamp with time zone,
+    FOREIGN KEY (open_content_provider_id) REFERENCES open_content_providers(id) ON UPDATE CASCADE ON DELETE CASCADE
+);
+CREATE INDEX idx_libraries_deleted_at ON public.libraries USING btree (id);
+CREATE INDEX idx_libraries_open_content_provider_id ON public.libraries USING btree (open_content_provider_id);
+
```

Comment:
can we make indexes on the URL and external_id here too? Only cuz its a migration and we cannot edit it later on without making a new one. then this is good to go :+1:

Reply:
@PThorpe92 These have been added.